specification.md

```diff
@@ -1631,6 +1631,15 @@ There are two types of conditions for switch states:
 
 These are exclusive, meaning that a switch state can define one or the other condition type, but not both.
 
+The order on which conditions are defined must be respected by implementations unless priorities are speficied. 
+If priorities are specified, the most prioritary among satisfied conditions must have its transition executed.
```

Comment:
Is "prioritary" a word? :) 
I would change this a little bit, for example, set up the issue up-front, something like:

At times multiple defined conditions can be evaluated to "true" by runtime implementations. This can lead to ambiguity on which condition and thus its transition the workflow control-flow logic should take next.
The specification has two ways of dealing with this ambiguity on the markup level:
1. Talk about the default order-based approach
2. Talk about the explicit approach using the priority parameter
3. Talk about how 1) and 2) are mutually exclusive, meaning its either one or the other (per switch state)

Hope this helps

Reply:
@tsurdilo I would say that if `priority` is not set, the order-based will take place for the remaining states.

Reply:
that would be confusing imo. in most cases I do believe that users will go with the default order-based ranking. We should imo not mix the two but make them mutually exclusive ways of defining priority.

Reply:
> Is "prioritary" a word? :)

I found some usages of it when googling, but I wasn't sure hehe. I'll change the wording as suggest and hopefully it will be simpler/clearer. :-)

> @tsurdilo I would say that if priority is not set, the order-based will take place for the remaining states.

I considered this approach, but we had previously talked about requiring all conditions to define priority (if one is defined) in #167. I see the value for UX in not requiring this, though with some complexity added for implementations.

I don't have a strong opinion regarding which way to go and would be fine with either, I'll send in changes once we decide which approach we're going to choose:

  a. require all conditions in a single state to have priorities if one of them has
  b. conditions with undefined priorities are checked first (in the order they appear) and then the ones which have priorities (ordered according to that)

Reply:
definitely 1)

Reply:
Well, I'm convinced. Let's make them mutually exclusive. If someone needs to control this, so they control each node.